MANIFEST.in

```diff
@@ -3,3 +3,5 @@ include *.txt
 recursive-include cf_units *.py
 include cf_units/etc/site.cfg.template
 include CHANGES COPYING COPYING.LESSER INSTALL
+include versioneer.py
+include cf_units/_version.py
```

Comment:
1. Isn't `cf_units/_version.py` already included by the earlier `recursive-include`?
2. Do we need `setup.cfg` listed in the manifest as well?